config/locales/en.yml

```diff
@@ -1238,37 +1219,24 @@ en:
       export_csv: "Export to CSV"
       email: "Email"
       approve: "Approve"
-      reject: "Reject"
-      delete: "Delete"
       waiting_list: "Waiting List"
       approved_registrations: "Approved registrations"
-      deleted_registrations: "Deleted registrations"
-      non_competing: "Non-competing registrations"
       not_paid: "Not paid"
       registered:
         without_stripe: "Registered on"
         with_stripe: "Paid on"
       #context: date is the registration creation date, eg: "2016-12-01 16:18:56 UTC"
       payment_requested_on: "Payment requested on: %{date}"
       payment_completed_on: "Payment completed on: %{date}"
-      #context: Counts how many parallel registrations in the same Series there are. Will display a number behind this label.
-      series_registrations: "Series registrations"
-      series_registrations_hint_html: "Hint: Series registrations are being displayed as <i>%{description}</i>"
       total: "Total"
-      n_events: "Number of events"
     new_registration:
       title: "Register for %{comp}"
-    registered_with_account_html: "This person registered with an account. You can edit their personal information %{here}."
     sign_in: "Sign in"
     profile: "profile"
     panel_title: "Registration details"
     register: "Register!"
-    refund: "Refund Payment"
-    refund_confirmation: "Are you sure you want to refund this payment?"
```

Comment:
Do we have a confirmation flow for refunds in the current system?

Reply:
done